collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2620,15 +2619,6 @@ public void onSaveFormIndexError(String errorMessage) {
 
     @Override
     public void onNumberPickerValueSelected(int widgetId, int value) {
```

Comment:
Huh. I don't know if we can just delete the code in here. Is that behaviour not needed any longer?

Reply:
I deleted it for this PR actually, since this concerns the range picker dialog. I had this back in Range Picker PR. is it fine?

Reply:
Ah. As much as possible we don't want to break things between PRs. Otherwise it means we can't release until the follow up PR comes in I now realize this PR actually removes the picker appearance. Is the "no-ticks" appearance gone as well? I think we should make sure to include all the Range widget work in this PR so we're not breaking anything when we merge.